### DISCUSSION TITLE
print instead of Print! Why not use snake_case for function naming convention instead of UpperCamelCase?

### DISCUSSION THREAD

**Discussion by muzam1l** [carbon-language/carbon-lang#1575] [Language design]
This would align it with cpp function naming convention and looks way prettier! 

Code importing cpp libraries will also look consistent in terms of function names, wonder what was the reason behind this decision when other common cpp design elements were bought in unmodified?

**Comment by jonmeow** [comment]
We're using different naming conventions than C++. See https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/naming_conventions.md.

That said, #1435 makes the argument towards lowercasing everything, I don't know what will come of it.
